services/user.js

```diff
@@ -114,6 +114,8 @@ class User {
     async #findStudiesNames(studies) {
         if (!studies) return [];
         const studiesIDs = (studies[0] instanceof Object) ? studies.map((study) => study?._id) : studies;
+        if(studiesIDs.includes("All"))
+            return ["All"];
```

Comment:
Can you change this logic? This return value is associated with the email notifications contents.

Reply:
What need to change?

Reply:
#findApprovedStudies will return intact approvedStudies including all data fields.
#findStudiesNames a convient util function to return studies names array for email.

Reply:
Peter, is your ticket or design associated with email notification contents? I am not familiar with the requirement for the **userRole change notification**. If we return **'All'**, this character will be included in the email contents.

Reply:
Yes.   need set value for studies in the email.  Let discuss with Ming to decide what value for "All".

Reply:
Updated to ["All studies"] if the input studies is ["All"]